Assets/Input System/Input Manager.cs

```diff
@@ -56,6 +58,8 @@ private void Awake()
         PickUpInteractor.OnBranchingPickup += BranchingItemPickedUp;
         playerInputActions.Branching.Navigate.performed += SwitchBranchingItem;
         playerInputActions.Branching.Submit.performed += SubmitBranchingItem;
+
+        inBranchingSelection = false;
```

Comment:
bit of a nitpicky thing, but you didn't need a new variable, you can just use `playerInputActions.Branching.enabled`